src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -174,6 +174,40 @@ public override string ReadLine()
             return ReadLine(false, string.Empty, out unused, true, true);
         }
 
+        /// <summary>
+        /// See base class
+        /// </summary>
+        /// <returns></returns>
+        /// <exception cref="HostException">
+        /// If obtaining a handle to the active screen buffer failed
+        ///    OR
+        ///    Win32's setting input buffer mode to disregard window and mouse input failed
+        ///    OR
+        ///    Win32's ReadConsole failed
+        /// </exception>
+        /// <exception cref="PipelineStoppedException">
+        /// If Ctrl-C is entered by user
+        /// </exception>
+        public override string ReadLineMaskedAsString()
+        {
+            HandleThrowOnReadAndPrompt();
+
+            const char printToken = '*'; // This is not localizable
```

Comment:
We can make this a class field and avoid defining it twice.

Reply:
fixed in 3fdcc5cb28823a4ffe38f702323fee76b270f293